shell/common/rasterizer.cc

```diff
@@ -670,6 +678,27 @@ std::optional<size_t> Rasterizer::GetResourceCacheMaxBytes() const {
   return std::nullopt;
 }
 
+bool Rasterizer::EnsureThreadsAreMerged() {
+  // If threads are merged statically, always return true.
+  if (task_runners_.GetRasterTaskRunner() ==
+      task_runners_.GetPlatformTaskRunner()) {
+    return true;
+  }
+  if (surface_ == nullptr || raster_thread_merger_.get() == nullptr) {
```

Comment:
in what case `surface_ = nullptr` ?

Reply:
If the rasterizer has already been torn down, the `surface_` would be null. And in that case we shouldn't try to merge threads or wait the threads to be merged.